appstream-change.patch

```diff
@@ -0,0 +1,56 @@
+diff --git a/org.kde.pix.appdata.xml b/org.kde.pix.appdata.xml
+index 3d66081..2ce1554 100644
+--- a/org.kde.pix.appdata.xml
++++ b/org.kde.pix.appdata.xml
+@@ -86,12 +86,12 @@
+     <p xml:lang="uk">Зручна програма для перегляду зображень і галерей.</p>
+     <p xml:lang="x-test">xxConvergent image and gallery viewer.xx</p>
+   </description>
+-  <url type="homepage">https://pix.kde.org/</url>
++  <url type="homepage">https://apps.kde.org/pix</url>
+   <url type="bugtracker">https://bugs.kde.org/enter_bug.cgi?product=pix</url>
+-  <url type="donation">https://www.kde.org/community/donations</url>
++  <url type="donation">https://kde.org/community/donations</url>
+   <screenshots>
+     <screenshot type="default">
+-      <image type="source" width="1440" height="720">
++      <image type="source" width="720" height="1440">
+           https://cdn.kde.org/screenshots/pix/pix.png
+       </image>
+       <caption>The Pix music player</caption>
+@@ -124,21 +124,33 @@
+       <caption xml:lang="zh-TW">Pix 音樂播放器</caption>
+     </screenshot>
+     <screenshot type="default">
+-      <image type="source" width="1440" height="720">
++      <image type="source" width="720" height="1440">
+           https://cdn.kde.org/screenshots/pix/pix-2.png
+       </image>
+     </screenshot>
+     <screenshot type="default">
+-      <image type="source" width="1440" height="720">
++      <image type="source" width="720" height="1440">
+           https://cdn.kde.org/screenshots/pix/pix-3.png
+       </image>
+     </screenshot>
+   </screenshots>
++  <recommends>
++    <control>pointing</control>
++    <control>keyboard</control>
++    <control>touch</control>
++  </recommends>
++  <content_rating type="oars-1.0" />
++  <categories>
```

Comment:
If the categories are defined in the desktop file, that should be enough. Both files are merged together later on

Reply:
This is good to know. I will remove it from the patch and from upstream. Thanks

Reply:
That happens by calling appstream-compose which caches the icons, screenshots & merges both informations on a single file that used by the stores later one :)